src/styles/book.css

```diff
@@ -0,0 +1,301 @@
+  opacity: 0.5;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.chap-X {
+  width: 13.25rem;
+  height: 1.313rem;
+  margin: 0.438rem 0 0.25rem 3.688rem;
+  font-family: 'Roboto Slab', serif;
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+  color: #121212;
+}
+
+.right-content {
+  display: flex;
+  position: absolute;
+  right: 15%;
+  margin-top: 40px;
+}
+
+.right-content li {
+  height: auto;
+}
+
+.progress-stat {
+  margin-right: 40px;
+  font-size: 25px;
+  margin-top: 15px;
+}
+
+.up-progress {
+  margin-top: 15px;
+  background-color: rgb(77, 77, 255);
+  padding: 10px;
+  border-radius: 5px;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: white;
+  height: 1.125rem;
+  width: 8.938rem;
+  margin-left: 60px;
+}
+
+.add-new-book {
+  width: 90%;
+  margin: auto;
+  margin-top: 40px;
+  height: 1.5rem;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+.inputs {
+  display: flex;
+  width: 90%;
+  margin: auto;
+  height: 2.813rem;
+}
+
+input {
+  height: 100%;
+  margin: 1.188rem 3.8rem 0 0;
+  border-radius: 4px;
+  background-color: var(--white-two);
+  border: 0.5px solid grey;
+}
+
+.book-name-input,
+.author-name-input {
+  width: 25%;
+}
+
+input::placeholder {
+  width: auto;
+  height: 1.188rem;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.15px;
+  color: rgb(171, 163, 165);
+  padding-left: 10px;
+}
+
+.category-selection {
+  width: 25%;
+  height: 100%;
+  margin: 1.188rem 2.125rem 0 -0.1rem;
+  padding: 0.375rem 0.938rem 0.563rem 1rem;
+  border-radius: 4px;
+  border: solid 0.5px grey;
+  background-color: var(--white-two);
+}
+
+.option {
+  width: 4.5rem;
+  height: 1.188rem;
+  margin: 0.438rem 10.438rem 0.25rem 0;
+  font-family: 'Montserrat', sans-serif;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.15px;
+  color: var(--very-light-pink);
+}
+
+.add-btn {
+  width: 15%;
+  height: 2.813rem;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.813rem;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: white;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
+  margin: 1.188rem 0 0 2.125rem;
+  border-radius: 3px;
+  background-color: rgb(77, 77, 255);
+}
```

Comment:
- [ ] The `background color` you have used for the buttons are not matching with the given design guideline. Kindly implement the given design guideline for styling. Please implement the color below for the background color of the button.
- ![image](https://github.com/ClintonYounge/My-Bookstore-Project/assets/60083437/9332020a-59a9-49c3-bd2e-860e12f347e2)